test/quoteSpec.js

```diff
@@ -126,6 +126,19 @@ describe('Quotes', function () {
         .end()
     })
 
+    it('should return a 422 if destination_ledger rounded amount is less than or equal to 0', function * () {
+      yield this.request()
+        .get('/quote?source_amount=0.00001' +
+          '&source_ledger=http://eur-ledger.example/EUR' +
+          '&destination_ledger=http://usd-ledger.example/USD')
+        .expect(422)
+        .expect(function (res) {
+          expect(res.body.id).to.equal('UnacceptableAmountError')
+          expect(res.body.message).to.equal('Quoted destination amount exceeds minimum amount allowed')
```

Comment:
Or "to go beyond a permitted limit". Do you have a suggestion for a
different word to use?
On Mon, Mar 28, 2016 at 5:44 PM Evan Schwartz <notifications@github.com>
wrote:

> In test/quoteSpec.js
> <https://github.com/interledger/five-bells-connector/pull/127#discussion_r57658070>
> :
>
> > @@ -126,6 +126,19 @@ describe('Quotes', function () {
> >          .end()
> >      })
> >
> > +    it('should return a 422 if destination_ledger rounded amount is less than or equal to 0', function * () {
> > +      yield this.request()
> > +        .get('/quote?source_amount=0.00001' +
> > +          '&source_ledger=http://eur-ledger.example/EUR' +
> > +          '&destination_ledger=http://usd-ledger.example/USD')
> > +        .expect(422)
> > +        .expect(function (res) {
> > +          expect(res.body.id).to.equal('UnacceptableAmountError')
> > +          expect(res.body.message).to.equal('Quoted destination amount exceeds minimum amount allowed')
>
> "Exceeds" means "is greater than"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/interledger/five-bells-connector/pull/127/files/02cedba5dacbe99e26bc7460d23232bfe8444d25..40bc7cbc747b72aaf29594b4dedd69611c26e2bc#r57658070>
>